### ISSUE TITLE
xorriso only create a combustion.iso when image base does not exists

### ISSUE THREAD

**Issue by alknopfler** [suse-edge/edge-image-builder#33]
Description:
- having this config (take a look is default without RT):
```
apiVersion: 1.0
image:
  imageType: iso
  baseImage: SLE-Micro.x86_64-5.5.0-Default-SelfInstall-GM.install.iso
  outputImageName: eib-image-slemicro55.iso
```
but using the image base with -RT for real time kernel into the images folder
```
$ ls
SLE-Micro.x86_64-5.5.0-Default-RT-SelfInstall-GM.install.iso
```
the script xorriso is creating the combustion successfully but adding to no image, so the script is not failing but it's not creating the final image.

I think a validation step is required to ensure the image present on the /images folder is the same than the config yaml file image.

**Comment by jdob** [comment]
Interesting, I'm surprised the xorriso command isn't blowing up. Or it is and I'm not getting all of the info out of the cmd in the code. I'll take a look.

**Comment by dbw7** [comment]
This is one of the validations things I tweaked earlier but I implemented poorly so I never put it up for PR. But yeah, the command continues whether or not the image file actually exists, so I think we just need a check to make sure the specified file/image can be found. Now down the line (or now, depending on what we think) we can look into actually checking if the image is an ISO/RAW for even more comprehensive validation, though this may not be necessary.
